tests/unit/test_middleware/test_base_authentication_middleware.py

```diff
@@ -44,7 +43,13 @@ class Auth:
 state: dict[str, AuthenticationResult] = {}
 
 
-class AuthMiddleware(AbstractAuthenticationMiddleware):
+class AuthMiddleware(ASGIAuthenticationMiddleware):
```

Comment:
We should still test `AbstractAuthenticationMiddleware`, as it wasn't removed

Reply:
done